RELEASE_NOTES.md

```diff
@@ -115,3 +115,4 @@ jdbc.pwd = ixmp
 - [#65](https://github.com/iiasa/message_ix/pull/65): Bugfix for downloading tutorials. Now downloads current installed version by default.
 - [#60](https://github.com/iiasa/message_ix/pull/60): Add basic ability to write and read model input to/from Excel
 - [#59](https://github.com/iiasa/message_ix/pull/59): Added MacOSX CI support
+- [#161](https://github.com/iiasa/message_ix/pull/161): A feature for adding new time steps to a scenario
```

Comment:
This line should go under the section for "Next Release", not for "v1.1.0 / Updates".